src/formulas/compiler.ts

```diff
@@ -169,7 +136,7 @@ export function compile(formula: string): CompiledFormula {
         }
       }
 
-      let listArgs: CompiledAST[] = [];
+      let listArgs: CodeBlock[] = [];
```

Comment:
a `listArgs` which is a list of `CodeBlock` seems weird (naming pov)

Reply:
listArgs => compiledArgs ?

Reply:
Seems better